tests/test_styles_cache.py

```diff
@@ -9,6 +10,8 @@
 from textual.geometry import Region, Size
 from textual.strip import Strip
 
+_CONSOLE = Console()
```

Comment:
Probably not a good idea to share this across tests. Don't see any particular issues, but we don't want to share state accross tests.

Suggest just using `Console()` in the tests.

Reply:
That was me trying to not waste time, since you recently told me `Console` does a fair bit of work that ends up being wasted in these tests.